src/Nox.Lib/Extensions/FluentApiExtensions.cs

```diff
@@ -15,4 +15,12 @@ public static IEnumerable<T> ForEach<T>(this IEnumerable<T> array, Action<T> act
             act(i);
         return array;
     }
+    public static bool HasAtLeastOneItem<T>(this List<T> list)
+    {
+        return list != null && list.Count > 0;
```

Comment:
Just need to find the first item
```suggestion
        return list != null && list.Count.Any();
```

Reply:
Done